python/docs/content_list.rst

```diff
@@ -1,39 +1,35 @@
 .. code-block:: bash
 
    Usage: nessie content list [OPTIONS]
-
+   
```

Comment:
Any idea why it's causing a diff in this file and log.rst?

Reply:
I believe this happens when you run tox locally